Formula/sxiv.rb

```diff
@@ -16,15 +17,16 @@ class Sxiv < Formula
   depends_on "giflib"
   depends_on "imlib2"
   depends_on "libexif"
-  depends_on :x11
+  depends_on "libx11"
+  depends_on "libxft"
 
   def install
     system "make", "PREFIX=#{prefix}", "AUTORELOAD=nop",
-                   "CPPFLAGS=-I/opt/X11/include", "LDFLAGS=-L/opt/X11/lib",
+                   "CPPFLAGS=-I#{HOMEBREW_PREFIX}/include/freetype2",
```

Comment:
Shouldn't this be something like 
```suggestion
                   "CPPFLAGS=-I#{Formula["freetype2"].opt_include}/freetype2",
```

Reply:
Good point. Fixing...